samples/wifi/provisioning/ble/sample.yaml

```diff
@@ -56,3 +56,19 @@ tests:
       - ci_build
       - sysbuild
       - ci_samples_wifi
+  sample.nrf7002eb2.ble-wifi-provision:
+    sysbuild: true
+    build_only: true
+    extra_args:
+      - ble_SHIELD="nrf7002eb2"
+      - ble_SNIPPET=nrf70-wifi
```

Comment:
Can you please add `ipc_radio_EXTRA_DTC_OVERLAY_FILE="../../samples/wifi/provisioning/ble/ipc_radio.overlay"` to the args? Without this the UART won't work.

Reply:
Need to add the file too:

```
$ cat samples/wifi/provisioning/ble/ipc_radio.overlay
/*
 * Copyright (c) 2025 Nordic Semiconductor ASA
 *
 * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 */

&uart135 {
    status = "disabled";
};
```

Reply:
And this is only for 54H, so, you need to split the entry.

Reply:
> And this is only for 54H, so, you need to split the entry.

Thanks for the info Chaitanya. As the `provisioning/ble` sample build is broken on 54H and is blocking other sample testing, it has been removed from this commit and will be addressed in another PR. I will handle this in that PR.